### DISCUSSION TITLE
Meshing Error

### DISCUSSION THREAD

**Discussion by jalfaro9** [Ifsttar/I-Simpa#347] [General]
Can someone help me with the following error:

preprocess - Mesh reparation has been aborted. The algorithm enter into an infinite loop. Try to stick coplanar faces or destroy manually.

After this, the program doesn't allow me to computize any calculation

**Comment by nicolas-f** [comment]
Hi.

Please provide the 3D model file in order to be able to help you

**Reply by ashley-gif** [reply]
Hi, I have the same problem with the following error which indicate preprocess - Mesh reparation has been aborted. The algorithm enter into an infinite loop. Try to stick coplanar faces or destroy manually. Can you please help me.
